src/a-runtime-console/kubernetes/model/pipelinestage.model.ts

```diff
@@ -43,10 +43,11 @@ export class PipelineStage {
             this.environmentName = name;
             let urlMap = se.serviceUrls;
             this.serviceUrlMap = urlMap;
-
-            let buildConfigName = build.buildConfigName;
-            if (buildConfigName) {
-              this.serviceUrl = urlMap[buildConfigName] || '';
+            let serviceUrlKey: string[] = Object.keys(urlMap);
```

Comment:
@invincibleJai Just a small suggestion. if you rename the variable from `serviceUrlKey` to `serviceUrlKeys`, it could be more meaningful.

Reply:
@jyasveer have made the changes